bowling.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='stylesheet' href='./css/style.css'>
+    <script src="./src/bowling.js"></script>
+    <script src="./src/frame.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src='./src/jquery.js'></script>
+    <title>Bowling Score Card</title>
+  </head>
+  <body>
+    <h1>Bowling Score Card</h1>
+    <div id="scorecard">
```

Comment:
Great usage of the score card number system. It makes it much easier to find which frame it is. Especially for situation like 'Strike'.